src/main/java/com/denizenscript/denizencore/scripts/ScriptLoggingContext.java

```diff
@@ -0,0 +1,70 @@
+package com.denizenscript.denizencore.scripts;
+
+import com.denizenscript.denizencore.DenizenCore;
+import com.denizenscript.denizencore.scripts.containers.ScriptContainer;
+import com.denizenscript.denizencore.scripts.containers.core.FormatScriptContainer;
+import com.denizenscript.denizencore.tags.ParseableTag;
+import com.denizenscript.denizencore.tags.TagManager;
+import com.denizenscript.denizencore.utilities.YamlConfiguration;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+
+public record ScriptLoggingContext(ParseableTag debugFormat, ParseableTag errorFormat) {
+    
+    // <--[language]
+    // @name Script Logging Format
+    // @group Script Container System
+    // @description
+    // Script logging contexts provide the format certain commands will use for their texts.
```

Comment:
This meta should probably have a `Most notably, this includes <@link command debug>`

Reply:
Added, and clarified the format a bit in general.